dist/component-war/src/main/webapp/WEB-INF/web.xml

```diff
@@ -37,7 +37,7 @@
 
   <resource-env-ref>
     <resource-env-ref-name>cache/publish</resource-env-ref-name>
-    <lookup-name>java:jboss/infinispan/cache/hawkular-metrics/publish</lookup-name>
+    <lookup-name>java:jboss/infinispan/cache/hawkular-alerts/publish</lookup-name>
```

Comment:
I thought I made the change everywhere the cache was used but I will look again.